lib/tasks/licenses.rake

```diff
@@ -4,7 +4,7 @@ namespace :licenses do  # rubocop:disable Metrics/BlockLength
   report_path = Rails.root.join('doc/licenses/licenses.xml')
 
   desc 'Generates a report with the dependencies and their licenses'
-  task :report do
+  task report: :environment do
```

Comment:
why?

Reply:
Rubocop changed it:
```
Offenses:

lib/tasks/licenses.rake:7:3: C: [Correctable] Rails/RakeEnvironment: Include :environment task as a dependency for all Rake tasks.
  task :report do
  ^^^^^^^^^^^^
```

Reply:
Disable that check. It is wrong. See https://github.com/3scale/porta/blob/master/Rakefile#L53-L60

Reply:
For these tasks also we absolutely don't need `environment`. It is very very strange for me that it tries to enforce a dependency that developer can see whether it is needed or not. It is a bad practice IMO to try loading everything for everything. What is the point of a dependency system then?

Reply:
OK

Reply:
Moreover rubocop should be bad practices check, while some people maybe think it has to be a reminder. I dunno why I'm so angry at this check.